krb5-testing-tools/README.md

```diff
@@ -0,0 +1,13 @@
+# krb5-testing-tools
+
+> [!CAUTION]
+> This image is only intended for Stackable-internal tests.
```

Comment:
We use it in demos too, right?

Reply:
Ah yeah... so it's name is kind of bad.
We'll think of an alternative name - otherwise I'll drop the PR.